students/kondel_tomasz/lesson_02_flow_control/the_second_maximum.py

```diff
@@ -0,0 +1,10 @@
+integer_list = []
+
+input_int = int(input())
+
+while input_int != 0:
+    integer_list.append(input_int)
+    input_int = int(input())
+
+integer_list.sort()
```

Comment:
This will work, but you will use O(n) memory and O(n * log(n)) time, which might be a bummer with larger data.